inbox/providers.py

```diff
@@ -412,6 +412,85 @@ def register_info_filter(self, name, func):
                        "mailstore1.asia.secureserver.net",
                        "mailstore1.europe.secureserver.net"]
     },
+    "163": {
+        "type": "generic",
+	"imap": ("imap.163.com", 993),
+	"smtp": ("smtp.163.com", 465),
+	"auth": "password",
+	"domains": ["163.com"],
+	"mx_servers": ["163mx00.mxmail.netease.com",
+			"163mx01.mxmail.netease.com",
```

Comment:
Could you align the strings on the same column?
For example, instead of this:
```
"mx_servers": ["163mx00.mxmail.netease.com",
   			    "163mx01.mxmail.netease.com",
```

the code should look like this:
```
"mx_servers": ["163mx00.mxmail.netease.com",
               "163mx01.mxmail.netease.com",
```
Thanks.